src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -380,6 +402,52 @@ return (
   language="jsx"
 />
 
+## TagSet Example
+
+<Canvas of={FilterBarInlineStories.TagSetExample} />
```

Comment:
Really good example. It feels a big laggy to me but do you notice it as well?

Reply:
I do notice some lag when updating the text input. The solution implemented for updating the tag set data is probably not the most efficient.

Reply:
Yes, it is a little laggy, but acceptable in this example.  I'm curious to see what happens as the number of input children increases.  Let's keep an eye on it.